anaconda_project/project.py

```diff
@@ -83,6 +83,13 @@ def _make_problems_into_objects(problems):
     return new_problems
 
 
+def _unknown_field_suggestions(project_file, problems, yaml_dict, known_fields):
+    for key in yaml_dict.keys():
+        if key not in known_fields:
+            problems.append(ProjectProblem(text="%s: Unknown field name '%s'" % (project_file.filename, key),
```

Comment:
ah ok. it's made a suggestion by the only_a_suggestion=True kwarg. Kind of confusing